fixing bug with getting frames of video

vsoch opened the pull request:
This fixes a bug I ran into on Ubuntu 18.04 with the default ffmpeg, and updated to 4.2, and I traced it down to imageio not correctly reading the number of frames (and throwing an error when it got to the last section and reporting 0). This comes down to changing:

```python
driving_video = imageio.mimread(opt.driving_video, memtest=False)
```

to

```python
driving_video = []
try:
    for im in reader:
        driving_video.append(im)
except RuntimeError:
    pass
```

I also:

 - added likely folders to the .gitignore
 - ensured we use the yaml.FullLoader for safe yaml loading

And I just want to say I'm absolutely in love with this work! I generated several videos using paintings over night, and am going to build a Docker -> Singularity container today to run on our cluster with GPU (and try out some of the other models!) Thank you so kindly for this work (and the spot on documentation for reproducing it!)

Signed-off-by: vsoch <vsochat@stanford.edu>

vsoch committed:
fixing bug with getting frames of video

Signed-off-by: vsoch <vsochat@stanford.edu>

AliaksandrSiarohin commented:
Thank you for your help.
1) Changing yaml loader not working with colab.
2) video, img, driving folders probably not relevant.

vsoch commented:
okay!

1. I've removed adding the extra files to the .dockerignore
2. I've removed the safe yaml loader (and I'd be curious to know why it doesn't work in colab, maybe a version conflict?)

The main change to fix the bug that I originally meant to address is still present.

AliaksandrSiarohin commented:
Thanks.
